.goreleaser.yaml

```
@@ -95,6 +95,9 @@ changelog:
       - '^docs:'
       - '^test:'
 
+signs:
+  - artifacts: checksum
```

```suggestion
  - artifacts: all
```

Would this do detached signatures?

---

The docs recommended using `checksum`, so thats what I went with here. It gives you transitive protection since the checksums would be signed and able to verify all other artifacts.